base_custom_info/README.rst

```diff
@@ -0,0 +1,288 @@
+    :target: https://runbot.odoo-community.org/runbot/149/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module allows you to attach custom information to records without the need
+to alter the database structure too much.
+
+This module defines several concepts that you have to understand.
+
+Templates
+---------
+
+A *template* is a collection of *properties* that a record should have.
+*Templates* always apply to a given model, and then you can choose among the
+current templates for the model you are using when you edit a record of that
+model.
+
+I.e., This addon includes a demo template called "Smart partners", that applies
+to the model ``res.partner``, so if you edit any partner, you can choose that
+template and get its properties autofilled.
+
+Properties
+----------
+
+A *property* is the "name" of the field. *Templates* can have any amount of
+*properties*, and when you apply a *template* to a record, it automatically
+gets all of its *properties* filled, empty (unless they have a *Default
+value*), ready to assign *values*.
+
+You can set a property to as *required* to force it have a value, although you
+should keep in mind that for yes/no properties, this would mean that only *yes*
+can be selected, and for numeric properties, zero would be forbidden.
+
+Also you can set *Minimum* and *Maximum* limits for every *property*, but those
+limits are only used when the data type is text (to constrain its length) or
+number. To skip this constraint, just set a maximum smaller than the minimum.
+
+*Properties* always belong to a template, and as such, to a model.
+
+*Properties* define the data type (text, number, yes/no...), and when the type
+is "Selection", then you can define what *options* are available.
+
+I.e., the "Smart partners" *template* has the following *properties*:
+
+- Name of his/her teacher
+- Amount of people that hates him/her for being so smart
+- Average note on all subjects
+- Does he/she believe he/she is the smartest person on earth?
+- What weaknesses does he/she have?
+
+When you set that template to any partner, you will then be able to fill these
+*properties* with *values*.
+
+Categories
+----------
+
+*Properties* can also belong to a *category*, which allows you to sort them in
+a logical way, and makes further development easier.
+
+For example, the ``website_sale_custom_info`` addon uses these to display a
+technical datasheet per product in your online shop, sorted and separated by
+category.
+
+You are not required to give a *category* to every *property*.
+
+Options
+-------
+
+When a *property*'s type is "Selection", then you define the *options*
+available, so the *value* must be one of these *options*.
+
+I.e., the "What weaknesses does he/she have?" *property* has some options:
+
+- Loves junk food
+- Needs videogames
+- Huge glasses
+
+The *value* will always be one of these.
+
+Recursive templates using options
```

Comment:
Is this feature enabled? I don't seem to find how to use it.

Reply:
I think that this refers to the feature of mix properties of different templates for a selection.

Reply:
Yes, but how do you use it? How do you specify in a selection property that links to a different template?

Reply:
Finally, it's an outdated readme... moved to roadmap. This feature it's not implemented yet. Thanks!